src/toolkit/pyroprocessing/pyre_winning.cc

```diff
@@ -0,0 +1,62 @@
+#include "pyre_winning.h"
+
+using cyclus::Material;
+using cyclus::Composition;
+using cyclus::CompMap;
+
+namespace pyro {
+
+Winning::Winning() {
+  SetCoeff();
+  current(0);
+  Rtime(0);
+  flowrate(0);
+  volume(0);
+}
+
+Winning::Winning(double new_current = 8, 
+                 double new_time = 1, 
+                 double new_flowrate = 3, 
+                 double new_volume = 1
+            ) 
```

Comment:
default value in the header ?

Reply:
with you actual implementation how do you know which Winning() constructor is actually called when calling Winning::Winning() ? (as the second one has 4 optional arguments ) (is that even compiling ?!??)

Reply:
I'm not sure what you mean, when it's constructed in pyre it is called using the 4 inputs so it is using the overloaded constructor (I do need to document the default constructor though).

The scenarios I've run have all received those 4 parameters so I'm fairly certain this is what happens. 

That being said though I can see some of the confusion as these default values are also given in the pragmas in pyre.h so are they really needed here? (or in the header as you suggest)

Reply:
This is not `pyre` class, this might be use by someone else in an other facility so I believe you should have the default value regardless of your usage in `pyre`

About the constructor,  I mean you have 2 constructor methods with the same name one without argument and one with 4 optional arguments (as they have default values), when you call the the constructor without arguments which one are you calling, the first one that set the value to `0` or the one using the default values ?